Userland/Libraries/LibPDF/Filter.cpp

```diff
@@ -206,16 +206,10 @@ PDFErrorOr<ByteBuffer> Filter::decode_png_prediction(Bytes bytes, int bytes_per_
     return decoded;
 }
 
-PDFErrorOr<ByteBuffer> Filter::decode_lzw(ReadonlyBytes)
+PDFErrorOr<ByteBuffer> Filter::handle_lzw_and_flate_parameters(ByteBuffer buffer, int predictor, int columns, int colors, int bits_per_component)
 {
```

Comment:
Consider adding `// Table 3.7 Optional parameters for LZWDecode and FlateDecode filters` as first line.